### ISSUE TITLE
example output

### ISSUE THREAD

**Issue by bl24** [michellejlin/covid_swift_pipeline#5]
Would it be possible to post the output for the example data? 

I ran the pipeline on the ex1 fastqs and I want to make sure I did it correctly. I'm looking at the clipped bam file with IGV and I'm seeing a lot of misalignments at the end of reads and reads that are stacking perfectly on top of each other. These are usually red flags for me when I look at alignments although I'll admit this is my first time working with viral data so I want to be sure if this is how the data is "supposed" to look.

**Comment by michellejlin** [comment]
Sure, I just uploaded the bam and fasta into example/ex1/.

I recently pushed a massive update to this master, which is now up to date (as most of the ongoing development is currently going on in the fork github.com/greninger-lab/covid_swift_pipeline). Did you run with the latest version? (You can do this by just adding `-latest` to your nextflow command). 

Here's a screenshot of what my IGV looks like for ex1 - 
![image](https://user-images.githubusercontent.com/42459461/109225383-0cd20680-7772-11eb-84d1-5ce4c4550da8.png)

The reads stacking perfectly on top of each other is pretty typical.

**Comment by bl24** [comment]
Great, thanks! 

I initially used which ever version existed on github.com/greninger-lab/covid_swift_pipeline on Feb22. I need to run the pipeline from a local copy because I need to edit the nextflow.config to run under the relatively limited power of my mac desktop. 

Today I redownloaded the pipeline from michellejlin/covid_swift_pipeline and I saw that it runs much faster than the version I used earlier this week. Also the variant output file is more informative.
